README.md

```diff
@@ -24,6 +24,16 @@ secrets:
   ENVIRONMENT_VAR:
       secretRef: some-secret-reference
       key: connectionString
+keyVaults:
```

Comment:
I wonder if this should be in a separate section, this won't work out of the box currently, as teams who need it will need the secret added into their namespace

maybe add that bit, thoughts?